vnext/Microsoft.ReactNative/Fabric/Composition/SwitchComponentView.cpp

```diff
@@ -0,0 +1,249 @@
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "SwitchComponentView.h"
+
+namespace Microsoft::ReactNative {
+
+SwitchComponentView::SwitchComponentView(
+    const winrt::Microsoft::ReactNative::Composition::ICompositionContext &compContext,
+    facebook::react::Tag tag,
+    winrt::Microsoft::ReactNative::ReactContext const &reactContext)
+    : Super(compContext, tag), m_context(reactContext) {
+  m_props = std::make_shared<facebook::react::SwitchProps const>();
+}
+
+std::vector<facebook::react::ComponentDescriptorProvider>
+SwitchComponentView::supplementalComponentDescriptorProviders() noexcept {
+  return {};
+}
+
+void SwitchComponentView::mountChildComponentView(const IComponentView &childComponentView, uint32_t index) noexcept {
+  assert(false);
+}
+
+void SwitchComponentView::unmountChildComponentView(const IComponentView &childComponentView, uint32_t index) noexcept {
+  assert(false);
+}
+
+void SwitchComponentView::handleCommand(std::string const &commandName, folly::dynamic const &arg) noexcept {
+  if (commandName == "setValue") {
+    // TODO
+  } else {
+    Super::handleCommand(commandName, arg);
+  }
+}
+
+void SwitchComponentView::updateProps(
+    facebook::react::Props::Shared const &props,
+    facebook::react::Props::Shared const &oldProps) noexcept {
+  const auto &oldViewProps = *std::static_pointer_cast<const facebook::react::SwitchProps>(m_props);
+  const auto &newViewProps = *std::static_pointer_cast<const facebook::react::SwitchProps>(props);
+
+  ensureVisual();
+
+  if (oldViewProps.backgroundColor != newViewProps.backgroundColor ||
+      oldViewProps.thumbTintColor != newViewProps.thumbTintColor || oldViewProps.value != newViewProps.value ||
+      oldViewProps.disabled != newViewProps.disabled) {
+    m_drawingSurface = nullptr;
+  }
+
+  m_props = std::static_pointer_cast<facebook::react::ViewProps const>(props);
+}
+
+void SwitchComponentView::updateState(
+    facebook::react::State::Shared const &state,
+    facebook::react::State::Shared const &oldState) noexcept {
+  // TODO?
```

Comment:
Doesn't look like Switch has any state right now, so I think you can just remove this.   I'm a little surprised that the value isn't in state, since it can be updated on both sides. But we can align with core here for now.